posthog/hogql_queries/experiments/trends_statistics_v2_count.py

```diff
@@ -36,9 +36,9 @@ def calculate_probabilities_v2_count(
     Returns:
     --------
     list[float]
-        A list of probabilities that sum to 1, where:
-        - The first element is the probability that the control variant is the best
-        - Subsequent elements are the probabilities that each test variant is the best
+        A list of probabilities where each element represents:
+        - index 0: probability control variant beats all test variants
```

Comment:
Yeah, control is a little tricky for this new behavior. Take this example:
* control and two variants, all equally good
* each variant will have a 50% chance to win (50/50 to beat control)
* control will only get a 33% chance to beat _both_ variants.

So one can no longer compare these values then. Control will look worse than it is. 

What I suggest we do, is that we for control show the probability to beat the _best_ variant. I think that will be more meaningful to the user. And maybe add a tool-tip that says "Probability to beat control" and "Probability to beat best variant", respectively.

Alternatively, we could also drop showing this value for control.